Rakefile

```diff
@@ -20,4 +20,7 @@ end
 # RSpec shoves itself into the default task without asking, which confuses the ordering.
 # https://github.com/rspec/rspec-rails/blob/eb3377bca425f0d74b9f510dbb53b2a161080016/lib/rspec/rails/tasks/rspec.rake#L6
 Rake::Task["default"].clear
-task default: [:lint, :spec, :test_unit, "jasmine:ci"]
+
+# Combined coverage only works if RSpec tests come last. Having two test frameworks sucks.
```

Comment:
```suggestion
# Combined coverage only works if RSpec tests come _after_ Minitest. Having two test frameworks sucks.
```

(`jasmine:ci` is the last task here so the comment is a little confusing!)

Reply:
Sure, I've amended that 👍